tests/project.bats

```diff
@@ -7,24 +7,24 @@ setup() {
 }
 
 @test "project init and update" {
-	cd "$TEMPDIR" || exit
+  cd "$TEMPDIR" || exit
```

Comment:
Why the indentation changes here?

Reply:
My IDEA use tab instead of space, I'll fix it